daisy_workflows/image_test/configuration/ubuntu.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python2
```

Comment:
And on the other hand, on rhel-6, there is no Python3.

I thought it would be easier to make a bootstrap script that actually installs python2 if we're already developing all scripts using python2.

It's hard to keep everything compatible, I agree. Please tell me if you have other thoughts.